src/test/java/com/trivago/fastutilconcurrentwrapper/longint/AbstractLongIntMapTest.java

```diff
@@ -0,0 +1,208 @@
+package com.trivago.fastutilconcurrentwrapper.longint;
+
+import com.trivago.fastutilconcurrentwrapper.AbstractMapTest;
+import com.trivago.fastutilconcurrentwrapper.LongIntMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+abstract class AbstractLongIntMapTest extends AbstractMapTest {
+    private LongIntMap map;
+    // Keep the default value to easily verify that this value is returned.
+    protected int defaultValue;
+    // Some methods return the default value of the underlying Fastutil implementation.
+    private static final int FASTUTIL_DEFAULT_VALUE = 0;
```

Comment:
a const should go on top
```suggestion
    // Some methods return the default value of the underlying Fastutil implementation.
    private static final int FASTUTIL_DEFAULT_VALUE = 0;
    
    private LongIntMap map;
    // Keep the default value to easily verify that this value is returned.
    protected int defaultValue;
```

Reply:
* Done